src/patternfly/components/Check/check-input.hbs

```diff
@@ -1,5 +1,21 @@
 <input class="{{pfv}}check__input{{#if check-input--modifier}} {{check-input--modifier}}{{/if}}"
   type="checkbox"
+  {{#ifAny check-description--text check--HasDescription check-body--text check--HasBodyText}}
+    aria-describedby="
+    {{#ifAny check-description--text check--HasDescription}}
+      {{check--id}}-description
+    {{/ifAny}}
+    {{#ifAny check-body--text check--HasBodyText}}
+      {{check--id}}-body
+    {{/ifAny}}"
```

Comment:
I wonder if we should...
1. Only have `aria-describedby` for either description if it exists, or a body if there is no description - instead of both.
2. Only use `aria-describedby` for the description, and not the body. The description should be something short/sweet and an actual description. The body could be a bunch of nested form controls or who knows what.